java-samples/corda5-negotiation-cordapp/workflows/src/main/java/com/r3/developers/samples/negotiation/workflows/modify/ModifyFlowArgs.java

```diff
@@ -0,0 +1,31 @@
+package com.r3.developers.samples.negotiation.workflows.modify;
+
+import net.corda.v5.base.annotations.CordaSerializable;
+
+import java.util.UUID;
+@CordaSerializable
+public class ModifyFlowArgs {
+    private UUID proposalID;
+    private int newAmount;
+
+    // use logs here instead??
+    //private ProgressMonitor progressMonitor = new ProgressMonitor()
```

Comment:
My opinion: I dont see the need to have loger in this class

Reply:
I agree, this is just a class for what the expected shape of the request to this workflow is.

Logging should be done in the `[Accept/Modify/Propose]FlowRequest` workflow.

I see that the responder flow just finalizes transactions, so it doesn't concern itself with the args class

Reply:
We spoke about this already Dhruv, but re-stating that ProgressTracking is now replaced with logging and const vals for each appropriately defined stage in the workflow